src/server/game/Spells/SpellEffects.cpp

```diff
@@ -799,15 +799,15 @@ void Spell::EffectDummy(SpellEffIndex effIndex)
 
 void Spell::EffectTriggerSpell(SpellEffIndex effIndex)
 {
-    if (effectHandleMode != SPELL_EFFECT_HANDLE_LAUNCH_TARGET
+    if (effectHandleMode != SPELL_EFFECT_HANDLE_HIT_TARGET
```

Comment:
I don't like that change - might break many spells using that effect. `SPELL_EFFECT_HANDLE_HIT_TARGET` might be `too late` hook for triggered spells.

Reply:
@grimgravy

Reply:
I didn't see a problem related to this

Reply:
There is a difference between launch and hit. And troggered spells must be launched with the base spell. In other case, all delayed spells would be broken.

Reply:
I will make a change soon.

Reply:
UltraNix@, this is very logical, isn't it? If it continues like this there is no solution.

Reply:
What is logical?

Reply:
@UltraNix, what is difference for you?

Reply:
`There is a difference between launch and hit. And triggered spells must be launched with the base spell. In other case, all delayed spells would be broken.`